src/DIRAC/TransformationSystem/Client/test/Test_Client_RequestTasks.py

```diff
@@ -0,0 +1,374 @@
+""" pytest for WorkflowTasks
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# pylint: disable=protected-access,missing-docstring,invalid-name
+import json
+import sys
+from mock import MagicMock
+
+from pytest import mark
+
+parametrize = mark.parametrize
+
+from hypothesis import given, settings, HealthCheck
+from hypothesis.strategies import (
+    composite,
+    integers,
+    lists,
+    text,
+    dictionaries,
+    from_regex,
+)
+from string import ascii_letters, digits
+
+# function_scoped_fixture is only used in Python 3 compatible release of hypothesis
+if sys.version_info.major >= 3:
+    function_scoped = (HealthCheck.function_scoped_fixture,)
+else:
+    function_scoped = tuple()
+
+
+from DIRAC.Core.Utilities.JEncode import encode
+
+
+from DIRAC.TransformationSystem.Client.BodyPlugin.DummyBody import DummyBody
+from DIRAC.TransformationSystem.Client.RequestTasks import RequestTasks
+
+
+@composite
+def taskStrategy(draw):
+    """Generate a strategy that returns a task dictionnary"""
```

Comment:
```suggestion
    """Generate a strategy that returns a task dictionary"""
```

Reply:
of course there's at least one :laughing:

Reply:
done